Code/Tools/ProjectManager/Source/AdjustableHeaderWidget.h

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#pragma once
+
+#if !defined(Q_MOC_RUN)
+#include <QTableWidget>
+
+#include <QHeaderView>
+#include <QStringList>
+#include <QVector>
+#include <QPair>
+#endif
+
+namespace O3DE::ProjectManager
+{
+    // Using a QTableWidget for its header
+    // Using a seperate model allows the setup of a header exactly as needed
+    class AdjustableHeaderWidget
+        : public QTableWidget
+    {
+        Q_OBJECT // AUTOMOC
```

Comment:
You don't need the `// AUTOMOC` anymore, you just need the `AUTOMOC` in the target in your CMakeLists.txt that includes this file (This comment applies to other files in this PR too)

Reply:
Removed AUTOMOC from all project manager header files